.github/workflows/integration_tests.yml

```diff
@@ -44,6 +44,10 @@ jobs:
             laravel: ^6.0
           - php: 8.2
             laravel: ^6.0
+          - php: 8.0
+            laravel: ^10.0
+          - php: 7.4
+            laravel: ^10.0
```

Comment:
Wondering why php 8 and 7.4 if laravel 10 min version is 8.1

Reply:
It's _excluding_ that combination.

Reply:
ok, makes sense now :smiley: